pkg/config/config.go

```diff
@@ -17,6 +17,7 @@ type Config struct {
 	Rules              []*rule.Rule `yaml:"rules"`
 	IgnoreFiles        []string     `yaml:"ignore_files"`
 	SuccessExitMessage *string      `yaml:"success_exit_message"`
+	AddNoteToMessage   bool         `yaml:"add_note_to_message"`
```

Comment:
I might prefer a simpler key, like `include_note` instead of `add_note_to_message`